spec/models/game_spec.rb

```diff
@@ -43,4 +43,33 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  describe "Populate Board" do
+    it "should have correct number of pieces" do
+      game = FactoryGirl.create(:game)
```

Comment:
There's no reason to create a new game for each test. You can simply do this once at the top of the `describe` block and then reference it in all the examples. Also, I've added an RSpec config option that simplifies factory_girl syntax, so you can use `create :game` in place of `FactoryGirl.create :game`.